x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -88,3 +99,60 @@ export function uniqueParentPidForProcess(passedEvent: ResolverEvent): string |
     return passedEvent.process.parent?.entity_id;
   }
 }
+
+/**
+ * Returns the process event's parent pid
+ */
+export function hostParentPidForProcess(passedEvent: ResolverEvent): string | undefined {
+  if (event.isLegacyEvent(passedEvent)) {
+    return String(passedEvent.endgame.unique_ppid);
+  } else {
+    const ppid = passedEvent.process.parent?.pid;
+    return ppid ? String(ppid) : undefined;
+  }
+}
+
+/**
+ * Returns the process event's path on its host
+ */
+export function hostPathForProcess(passedEvent: ResolverEvent): string | undefined {
+  if (event.isLegacyEvent(passedEvent)) {
+    return passedEvent.endgame.process_path;
+  } else {
+    return passedEvent.process.executable;
+  }
+}
+
+/**
+ * Returns the username for the account that ran the process
+ */
+export function userInfoForProcess(passedEvent: ResolverEvent): object | undefined {
```

Comment:
nit: not sure if it'd be helpful to define an interface for the actual return type.

Reply:
sure doesn't hurt. thanks for the spot: https://github.com/elastic/kibana/pull/69098/commits/499d9429daad35c88fea0e9945f2dd780e910c58